js/enquiry.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function(){
+	if($('#enquiry-form').show()){
+		$('.fakeUpload').on('click',function (){
+			$('#file').click();
+		});
+		$('#file').on('change',function(){
+			var filename = this.value;
+		    var lastIndex = filename.lastIndexOf("\\");
+		    if (lastIndex >= 0)
+		        filename = filename.substring(lastIndex + 1);
+	    	$('#filename').html(filename);
+		});
+		 $('.submitForm').on('click', function(){
+		 	var checkedAtLeastOne = false;
+			$('input[type="checkbox"]').each(function() {
+			    if ($(this).is(":checked")) 
+			        checkedAtLeastOne = true;
+			});
+			if(!checkedAtLeastOne)
+				$('#checkbox1').prop("checked",true);
+			$('#submit').click();
```

Comment:
Ok, no problem. Thanks a lot, Kit.:)

There are some copywriting issues need your help too. For all copywriting issue, we can do all of them in one commit. Let's talk and revise them next Monday! Have a nice weekend! :D